framework/include/timesteppers/Stepper.h

```diff
@@ -0,0 +1,606 @@
+#pragma once
+
+#define MAKETOGETHER(a,b) a##b
+#define DBG MAKETOGETHER(std::c,out)
+
+#include <functional>
+
+#include "LinearInterpolation.h"
+#include "libmesh/numeric_vector.h"
+
+class Logger
```

Comment:
Is this meant just to log `Stepper`s?  If so, put it inside `Stepper`.  Right now it's polluting the global namespace with `Logger`